week3/todoCLI.js

```diff
@@ -0,0 +1,115 @@
+const readline = require('readline');
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+let myStuff = []; 
+//The above "myStuff" array will be storing the todo's that a user enters.
+function userInput(answer){
+    if (answer === 'v'){
+        viewList();
+    } else if (answer === 'n'){
+        addNewItem();
+    } else if (answer[0] === 'c') {
+        completeItem(answer);
+    } else if (answer[0] === 'd') {
+        deleteItem(answer);
+    } else if (answer === 'q'){
+        quitList();
+    } else {
+        console.log(`You must enter one of the following: v, n, cX (where X is a number), dX (where X is a number), q`);
+        menu();
+    };
+};
+
+function menu () {rl.question(
+    `Welcome to ToDo CLI!\n--------------------\n(v) View ∙ (n) New ∙ (cX) Complete ∙ (dX) Delete ∙ (q) Quit\n>`, 
+    userInput)};
+    menu();
```

Comment:
should not be indented if you are calling this now. Otherwise it gives the impression that this should be/currently is within another function